### ISSUE TITLE
Can't right-click on blue underline struggled words.

### ISSUE THREAD

**Issue by Gladox114** [languagetool-org/languagetool#6970]
Can't right-click on blue underline struggled words because this error sometimes pops out. Or when checking for next spelling mistake, or when pressing F7 to check text.

![image](https://user-images.githubusercontent.com/53704522/181805506-12b1e998-8f1b-4a7e-84cb-a29df2600552.png)

Using LibreOffice 7.3.4.2 30(Build:2)
with LanguageTool 5.7

**Comment by Gladox114** [comment]
I forgot to mention that it's not always the case, and I am not sure why it's happening either. But in my current document it happens 90% of the time

**Comment by Tex2002ans** [comment]
Can you download the latest LanguageTool 5.8 and see if it fixes your problem?

**Comment by Gladox114** [comment]
After opening my file, I get this error message only one time.
![image](https://user-images.githubusercontent.com/53704522/182044618-dd894b12-75ba-4be1-a7d8-69c199e1c4b5.png)

Then after right-clicking on Blue Struggled words, I always get this.
![image](https://user-images.githubusercontent.com/53704522/182044660-6af43270-6580-4c5f-9548-d97e86377256.png)

**Comment by FredKruse** [comment]
This problem should be solved by the next LT snapshot. Please download the latest OXT, after tomorrow (2022-08-05), at:
https://internal1.languagetool.org/snapshots/
Please test it with your file and give a comment.

**Comment by Gladox114** [comment]
When opening my file I get this error.

![image](https://user-images.githubusercontent.com/53704522/183309137-2362610b-90c6-44c9-869c-839695b18cb4.png)

**Comment by Gladox114** [comment]
I managed to create this error message again on my Laptop with the 5.7 version. I updated to snapshot 20220809, and suddenly it isn't creating any error messages anymore when right-clicking on blue struggled words. Furthermore, I hope this is fixed as I constantly change my document and forgot to keep a backup for this issue. But I guess this situation should be enough evidence that it finally works
